R/theme_cmap.R

```diff
@@ -275,7 +278,7 @@ theme_cmap_base <- function(consts = cmapplot_globals$consts,
     panel.background =   NULL,
     panel.border =       element_blank(),
     panel.grid =         element_blank(),
-    panel.spacing =      unit(consts$half_line, "pt"),
+    panel.spacing =      unit(consts$half_line * 2, "pt"),
```

Comment:
Is there a reason we are using points here, and not big points? I understand why the legend key is being done in points, but I'm not sure about this one.

Reply:
a reasonable argument. Technically, the `half_line` comes from the font size, which is why these were all in `pt`. But, it does seem reasonable to adjust them all to `bigpts`. I'll adjust now.